Makefile

```diff
@@ -36,7 +36,7 @@ test: build
 
 .PHONY: schema_doc
 schema_doc:
-	# https://github.com/coveooss/json-schema-for-humans
+	# https://github.com/coveooss/json-schema-for-humans v0.45.2
```

Comment:
This was the version that appears to have been used to generate the file the last time.
Using newer versions causes other diffs, since the tool has changed its output format a bit.